proto/grpc/federation/federation.proto

```diff
@@ -179,6 +179,8 @@ message CallExpr {
   optional string timeout = 3;
   // retry specifies the retry policy if the method call fails.
   optional RetryPolicy retry = 4;
+  // header configures the response header metadata.
+  repeated HeaderConfig header = 5;
```

Comment:
Should I name it `headers`? I remember we talked about it before and went with a singular field name at that time.

Reply:
`header` is good 👍